packages/flutter/lib/src/material/app.dart

```diff
@@ -171,6 +173,31 @@ class MaterialApp extends StatefulWidget {
 
   /// {@macro flutter.widgets.widgetsApp.localizationsDelegates}
   ///
+  /// Internationalized apps that require translations for one of the locales
+  /// listed in [GlobalMaterialLocalizations] should specify this paramter
+  /// and list the [supportedLocales] that the application can handle.
+  ///
+  /// ```dart
```

Comment:
Add a `## Sample Code` heading to get our analyzer checks to run over this code sample?

Reply:
I don't think that would work with code that depended on the flutter_localizations package.